openshift/README.md

```diff
@@ -31,7 +31,7 @@ $ kubectl create namespace kyverno
 **Install the Kyverno chart:**
 
 ```console
-$ helm install kyverno --namespace kyverno kyverno ./kyverno-charts/nirmata
+$ helm install kyverno --namespace kyverno kyverno ./kyverno-charts/nirmata --set licenseManager.licenseKey=<license key obtained from Nirmata support or SAAS>,licenseManager.apiKey=<api key obtained from Nirmata support or SAAS>
```

Comment:
Add a prerequisite section to get the licensekey and apikey from support@nirmata.com

Reply:
Changed